packages/extension-api/src/extension-api.d.ts

```diff
@@ -1953,13 +1961,9 @@ declare module '@podman-desktop/api' {
     export function stopContainer(engineId: string, id: string): Promise<void>;
     export function deleteContainer(engineId: string, id: string): Promise<void>;
     export function buildImage(
-      containerBuildContextDirectory: string,
-      relativeContainerfilePath: string,
-      imageName: string,
-      platform: string,
-      selectedProvider: ProviderContainerConnectionInfo | containerDesktopAPI.ContainerProviderConnection,
+      context: string,
       eventCollect: (eventName: 'stream' | 'error' | 'finish', data: string) => void,
-      abortController?: AbortController,
+      options?: BuildImageOptions,
```

Comment:
it requires at least a comment that if provider is not provided, it will try with the first running provider and with a preference of a provider using podman engine type